src/server/debugcmd.cc

```diff
@@ -161,26 +161,31 @@ void DoPopulateBatch(string_view type, string_view prefix, size_t val_size, bool
   absl::InsecureBitGen gen;
   for (unsigned i = 0; i < batch.sz; ++i) {
     string key = absl::StrCat(prefix, ":", batch.index[i]);
+    int32_t elements_left = elements;
```

Comment:
I think we can add a test which checks that the peak RSS memory after `DEBUG POPULATE` command doesn’t significantly differ from the rss usage.